game/kernel/kmemcard.cpp

```diff
@@ -100,12 +100,12 @@ void mc_print(const std::string& str, Args&&... args) {
 }
 
 const char* filename[12] = {
-    "/BASCUS-97124AYBABTU!",           "/BASCUS-97124AYBABTU!/icon.sys",
-    "/BASCUS-97124AYBABTU!/icon.ico",  "/BASCUS-97124AYBABTU!/BASCUS-97124AYBABTU!",
-    "/BASCUS-97124AYBABTU!/bank0.bin", "/BASCUS-97124AYBABTU!/bank1.bin",
-    "/BASCUS-97124AYBABTU!/bank2.bin", "/BASCUS-97124AYBABTU!/bank3.bin",
-    "/BASCUS-97124AYBABTU!/bank4.bin", "/BASCUS-97124AYBABTU!/bank5.bin",
-    "/BASCUS-97124AYBABTU!/bank6.bin", "/BASCUS-97124AYBABTU!/bank7.bin"};
+    "BASCUS-97124AYBABTU!",           "BASCUS-97124AYBABTU!/icon.sys",
+    "BASCUS-97124AYBABTU!/icon.ico",  "BASCUS-97124AYBABTU!/BASCUS-97124AYBABTU!",
+    "BASCUS-97124AYBABTU!/bank0.bin", "BASCUS-97124AYBABTU!/bank1.bin",
+    "BASCUS-97124AYBABTU!/bank2.bin", "BASCUS-97124AYBABTU!/bank3.bin",
+    "BASCUS-97124AYBABTU!/bank4.bin", "BASCUS-97124AYBABTU!/bank5.bin",
+    "BASCUS-97124AYBABTU!/bank6.bin", "BASCUS-97124AYBABTU!/bank7.bin"};
```

Comment:
Does the leading backslash cause issues? Don't really think it's a big deal to change these strings, but it would be nice if our filepath stuff could handle redundant slashes properly.

Reply:
When being combined with `std::filesystem` it's a problem because you end up with a path with two slashes.  String concatenation of course avoids this.  We could add extra code and steps to handle duplicate slashes but, we should probably just be constructing paths correctly and not have to go through another abstraction layer.